File: java/src/projects/PigLatin.java
package projects;

public class PigLatin {
	public static void main(String[] args) {
		PigLatin("Banana");
		PigLatin("example");
	}
	private static void PigLatin(String inputString) {
		char[] consonants = {'b', 'c', 'd', 'f', 'g', 'h', 'j', 'k', 'l', 'm', 'n', 'p', 'q', 'r', 's', 't', 'v', 'w', 'x', 'y', 'z'};
		char firstChar = inputString.toLowerCase().charAt(0);
		String pigLatin = "";
		Boolean consonantFirst = false;
		for (char consonant : consonants) {
			if(firstChar == consonant){
				pigLatin = inputString.replaceFirst(String.valueOf(inputString.charAt(0)), "") + consonant + "ay";
				System.out.println(pigLatin);
				consonantFirst = true;
				break;
			}
		}
		if(consonantFirst == false){
			pigLatin = inputString + "way";
			System.out.println(pigLatin);
		}
	}
}

File: python/pig_latin.py
def pig_latin(string):
	string = string.lower()
	consonants = ['b', 'c', 'd', 'f', 'g', 'h', 'j', 'k', 'l', 'm', 'n', 'p', 'q', 'r', 's', 't', 'v', 'w', 'x', 'y', 'z']
	first_letter = string[0]
	pig_latin = ''
	for consonant in consonants:
		if first_letter == consonant:
			pig_latin = string.split(consonant, 1)[1] + first_letter +"ay"
			return pig_latin
	else:
		pig_latin = string + "way"
		return pig_latin

print pig_latin("Banana")
print pig_latin("example")